src/main/java/com/duckblade/osrs/dpscalc/ui/DpsPluginPanel.java

```diff
@@ -119,4 +121,23 @@ public void openNpcStats(NpcStats preload)
 		currentPanel.openNpcStats(preload);
 	}
 
+	public void onEquipmentChanged()
+	{
+		inputPanels.forEach(DpsCalcPanel::onEquipmentChanged);
```

Comment:
These proxy methods can be removed, we can only update the current set, since without this limitation we effectively make all sets equivalent on every change.